Add guidance for authorizing a Facebook account (#5651)
segmentio/segment-docs: src/connections/destinations/catalog/personas-facebook-custom-audiences/index.md

### Not seeing an audience in Facebook

> If the syncs to the destination are failing, this could be due to an authorization error. Whoever created the destination account will need to be the one who accepts the TOS. The account manager will then need to log in to their Facebook account > Audiences > Search Audience > Click 'Accept Terms'.

Reviewer: ```suggestion
If syncs to the destination are failing, this might be due to an authorization error. Whoever created the destination account needs to accept the TOS. The account manager then needs to log in to their Facebook account, navigate to **Audiences > Search Audience** and click **Accept Terms**.
```
